src/main.py

```diff
@@ -0,0 +1,29 @@
+from s2_fos import S2FOS
```

Comment:
What's the purpose of this file?

Reply:
It is same example code as in the README.md I will remove it.